packages/ocaml_intrinsics/ocaml_intrinsics.v0.15.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "Jane Street developers"
+authors: ["Jane Street Group, LLC"]
+homepage: "https://github.com/janestreet/ocaml_intrinsics"
+bug-reports: "https://github.com/janestreet/ocaml_intrinsics/issues"
+dev-repo: "git+https://github.com/janestreet/ocaml_intrinsics.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/ocaml_intrinsics/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"             {>= "4.08.0"}
+  "dune"              {>= "2.0.0"}
+  "dune-configurator"
+]
+available: (arch = "x86_32" | arch = "x86_64") & os != "win32"
```

Comment:
Does it also work on macOS/arm64?
```suggestion
available: (arch = "x86_32" | arch = "x86_64" | arch = "arm64") & os != "win32"
```

Reply:
I haven't been able to test it yet on amcOS/arm64, I don't have access to one. Can we fix it for x86 first, or do you prefer to wait until arm64 is fixed and tested?

Reply:
If it helps, i've been using `ocaml_intrinsics` on an arm64 mac (m1) with the patch version https://github.com/janestreet/ocaml_intrinsics/commit/7d7668a753c1ada0d1d79e7e1fa6a3eddb61d349

I haven't noticed any issues and the tests from https://github.com/yallop/intrinsics-report pass on the m1 mac.